scripts/shared/lib/deploy_helm

```diff
@@ -26,8 +26,8 @@ function setup_broker() {
     fi
 
     submariner_broker_url=$(kubectl -n default get endpoints kubernetes -o jsonpath="{.subsets[0].addresses[0].ip}:{.subsets[0].ports[?(@.name=='https')].port}")
-    submariner_broker_ca=$(kubectl -n "${BROKER_NAMESPACE}" get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='${BROKER_CLIENT_SA}')].data['ca\.crt']}")
-    submariner_broker_token=$(kubectl -n "${BROKER_NAMESPACE}" get secrets -o jsonpath="{.items[?(@.metadata.annotations['kubernetes\.io/service-account\.name']=='${BROKER_CLIENT_SA}')].data.token}"|base64 --decode)
+    submariner_broker_ca=$(kubectl -n "${BROKER_NAMESPACE}" get secrets "${BROKER_NAMESPACE}-client-token" -o jsonpath="{.data['ca\.crt']}")
+    submariner_broker_token=$(kubectl -n "${BROKER_NAMESPACE}" get secrets "${BROKER_NAMESPACE}-client-token" -o jsonpath="{.data.token}"|base64 --decode)
```

Comment:
shouldn't this be `${BROKER_CLIENT_SA}-client-token`?

Reply:
`${BROKER_CLIENT_SA}-token`, perhaps — that would be more legible, but the result is ultimately the same (`BROKER_CLIENT_SA` is effectively `${BROKER_NAMESPACE}-client`).